src/test/java/uk/gov/digital/ho/pttg/testrunner/ComponentHeaderCheckerTest.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.digital.ho.pttg.testrunner;
+
+import org.junit.Test;
+
+import java.util.List;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ComponentHeaderCheckerTest {
```

Comment:
Could add a test to show that whitespace in a single list of multiple headers doesn't matter.

Reply:
Good idea, I'll add it.